frontend/src/components/Status.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+    <div> 
+        <span
+        :class="['tag', stat.style]"
+        > 
+        <li>{{stat.label}}</li>
+        </span>      
+    </div>
+</template>
+
+<script>
+export default {
+    name: "Status",
+    data(){
+        return {
+            statuses: [ 
+            {"label": "Screening", "value": "SCREENING", "style": "is-info" },
+            {"label": "Unmatched", "value": "UNMATCHED", "style": "is-warning"},
+            {"label": "Matched", "value": "MATCHED", "style": "is-success"},
+            {"label": "Dropped Out", "value": "DROPPED OUT", "style": "is-danger"}
+            ]
+        }
+    },
+    props: ["status"],
+    computed: {
+        stat() {
+            return this.statuses.find(item => item.value === this.status)
```

Comment:
`find` can return `undefined`, if the thing you are looking for does not exist. Since we are accessing properties of the "found" item (like `stat.style`), it would be safer to pass a default in case nothing is found.
```suggestion
            return this.statuses.find(item => item.value === this.status) || { label: "", value: "", style: "" }
```

Reply:
roger that! thanks for the suggestion :D